cores/esp8266/core_esp8266_si2c.cpp

```diff
@@ -422,13 +428,17 @@ unsigned char Twi::readFrom(unsigned char address, unsigned char* buf, unsigned
     {
         write_stop();
     }
+    else
+    {
+        twi_scl_valley();
+        WAIT_CLOCK_STRETCH();
+        // TD-er: Also busywait(twi_dcount) here?
+        // busywait(twi_dcount);
+    }
     i = 0;
     while (!SDA_READ() && (i++) < 10)
     {
-        SCL_LOW();
-        busywait(twi_dcount);
-        SCL_HIGH();
-        WAIT_CLOCK_STRETCH();
```

Comment:
Clock stretching is allowed everywhere in normal Fast/Fast+ mode (<1MHz).  https://www.nxp.com/docs/en/user-guide/UM10204.pdf

Only HS mode (up to 3.4MHz) doesn't allow clock stretching, however the code itself only takes args up to 400KHz.

So, can we please re-add the WAIT_CLOCK_STRETCH() back here?

Reply:
Sure, will add it.